### DISCUSSION TITLE
Welcome to rswag Discussions!

### DISCUSSION THREAD

**Discussion by BookOfGreg** [rswag/rswag#494] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BookOfGreg** [comment]
Hi I'm @BookOfGreg, DomainDrivenDev created the Rswag project and I joined DDD to help keep the lights on, keep deployments flowing, and help arrange the effort others are putting in. Help and encouragement is always welcome! The more the merrier and many hands truly to make light work.
I'm doing this because I love the ruby community, I love API tooling such at Swagger and Pact, and it's great to be able to contribute back to keeping this vibrant and productive for us all.

**Comment by jtannas** [comment]
Hi I'm @jtannas and I joined around the same time as `@BookOfGreg` to help keep this project going. While writing this I'm still learning the code, so for now I'm focusing on the triage side of things.

It was a complete lifesaver when I had to replicate an older API from just the swagger documentation. We fed the old docs into a script and a template, then out popped ~1200 RSwag tests that would've taken us weeks to write by hand in rspec. I figure anything that powerful at testing & documentation is worth the effort to keep alive.
